lib/LambdaHelper.js

```diff
@@ -152,4 +232,57 @@ module.exports = function (aws, queueInitializedPromise, executionStore){
 
         return proxyStore[functionIdentifier];
     };
+
+    this.getCloudedFunctionFromModule = function(modulePath, handlerName, deps, configs) {
```

Comment:
I think this would be easier to use if the params were in an object, no need to stick to a specific order ...
I think it would be easier to handle params as well.. Let's say I don't have any deps and I'd like to specify some configs i have to  call it like ``` λ(require.resolve('./calculator'), 'add', [], { name: 'ADD', memory: 256 }); ``` which i think suck

I would like to call it like  
```
var params = {
    handlerName: 'add',
    lamdaConfigs: {
        name: 'ADD',
        memory : '256'
    }
}
λ(require.resolve('./calculator'), params);
```